docs/api/networksocket/DTLSSocket.md

```diff
@@ -10,6 +10,10 @@ To use secure DTLS connections, application use the `DTLSSocketWrapper` through
 
 [![View code](https://www.mbed.com/embed/?type=library)](https://os.mbed.com/docs/development/mbed-os-api-doxy/class_d_t_l_s_socket.html)
 
+### DTLSSocket example
+
+For example, please see the TLSSocket socket example.
```

Comment:
~for example,~ Please see the [TLSSocket example](link?).

Not sure how to link to examples, sorry.